src/shared/sam/debugger/commands/addSamDebugConfiguration.ts

```diff
@@ -79,18 +78,13 @@ export async function addSamDebugConfiguration(
                 }
             }
         }
-        samDebugConfig = createTemplateAwsSamDebugConfig(
-            resourceName,
-            workspaceFolder ? getNormalizedRelativePath(workspaceFolder.uri.fsPath, rootUri.fsPath) : rootUri.fsPath,
-            preloadedConfig
-        )
+        samDebugConfig = createTemplateAwsSamDebugConfig(workspaceFolder, resourceName, rootUri.fsPath, preloadedConfig)
     } else if (type === CODE_TARGET_TYPE) {
         // strip the manifest's URI to the manifest's dir here. More reliable to do this here than converting back and forth between URI/string up the chain.
         samDebugConfig = createCodeAwsSamDebugConfig(
+            workspaceFolder,
             resourceName,
-            workspaceFolder
-                ? getNormalizedRelativePath(workspaceFolder.uri.fsPath, path.dirname(rootUri.fsPath))
-                : path.dirname(rootUri.fsPath),
+            path.dirname(rootUri.fsPath),
```

Comment:
`addSamDebugConfiguration()` covers the codelens cases, but not the case described in the PR description